re.c

```diff
@@ -2373,21 +2373,26 @@ match_inspect_name_iter(const OnigUChar *name, const OnigUChar *name_end,
 
 /*
  * call-seq:
- *    mtch.inspect   -> str
+ *   inspect -> string
+ *
+ * Returns a string representation of +self+:
  *
- * Returns a printable version of <i>mtch</i>.
+ *    m /.$/.match("foo")
+ *    m.inspect # => "#<MatchData \"o\">"
  *
- *     puts /.$/.match("foo").inspect
- *     #=> #<MatchData "o">
+ *    /(.)(.)(.)/.match("foo")
+ *    # => #<MatchData "foo" 1:"f" 2:"o" 3:"o">
+ *    m.inspect # => "#<MatchData \"o\">"
```

Comment:
This line and other `m.inspect` lines below look wrong.

Reply:
Fixed.